domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -2271,6 +2271,184 @@ public void getDistinctOnWithFacets(TestContext context) throws IOException {
     async2.awaitSuccess();
   }
 
+  @Test
+  public void getCQLWrapperFailure(TestContext context) throws IOException, FieldException {
+    final String tableDefiniton = "id UUID PRIMARY KEY , jsonb JSONB NOT NULL, distinct_test_field TEXT";
+    postgresClient = createTableWithPoLines(context, MOCK_POLINES_TABLE, tableDefiniton);
+
+    CQL2PgJSON cql2pgJson = new CQL2PgJSON("jsonb");
+    {
+      CQLWrapper cqlWrapper = new CQLWrapper(cql2pgJson,
+        "cql.allRecords="); // syntax error
+      Async async = context.async();
+      postgresClient.get(MOCK_POLINES_TABLE, Object.class, "*",
+        cqlWrapper, true, true, false, null, null/*facets*/, handler -> {
+          context.assertTrue(handler.failed());
+          async.complete();
+        });
+      async.awaitSuccess();
+    }
+  }
```

Comment:
This can be reduced to

    @Test
    public void getCQLWrapperFailure(TestContext context) throws Exception {
      CQL2PgJSON cql2pgJson = new CQL2PgJSON("jsonb");
      CQLWrapper cqlWrapper = new CQLWrapper(cql2pgJson,
          "cql.allRecords="); // syntax error
      postgresClient().get(MOCK_POLINES_TABLE, Object.class, "*",
          cqlWrapper, true, true, false, null, null/*facets*/, context.asyncAssertFailure());
    }